pkg/cmd/clustertriggerbinding/clustertriggerbinidng_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
Just a question. Is this related to this pr?

Reply:
I was going through tests, and I found this is missing one and there are few redundant golden files which are not used, so did all refactoring :D